app/src/org/commcare/models/AndroidSessionWrapper.java

```diff
@@ -198,7 +199,12 @@ public void commitStub() {
         setFormRecordId(r.getID());
 
         SessionStateDescriptor ssd = SessionStateDescriptor.buildFromSessionWrapper(this);
-        sessionStorage.write(ssd);
+        try {
+            sessionStorage.write(ssd);
+        } catch (Exception e) {
+            CrashUtil.log("SessionStateDescriptor form id: " + ssd.getFormRecordId());
```

Comment:
Just a heads up that if there's ~any chance SSD can be null here that this will actually obfuscate the underlying error by resulting in a null pointer instead.